space_weather/ace_invalid_spec.csh

```diff
@@ -16,8 +16,8 @@ set lock = "/pool1/mta/prot_spec_violate"
       echo "see http://cxc.harvard.edu/mta/ace.html" >> $lock
 
       echo "This message sent to sot_yellow_alert" >> $lock
-      cat $lock | mailx -s ACE_p5/p6 sot_yellow_alert
-      #cat $lock | mailx -s ACE_p5/p6 brad swolk
+      cat $lock | mailx -s ACE_p5/p6 sot_yellow_alert@cfa.harvard.edu
```

Comment:
I see when we get to the csh files that the @ symbols aren't escaped.  That should be fine (based on my cmdline test) but I see that ace_invalid_data.csh (which is maybe unused?) escapes them.  So should probably double-check this is what you want.

Reply:
Both escaped and not escaped versions work based on my cmd line test. Both ace_invalid_data.csh and ace_invalid_spec.csh are not used, as far as I can tell.